src/app/SERVICE/tasks.service.ts

```diff
@@ -1,61 +1,77 @@
 import { Injectable } from '@angular/core';
-import { Task } from '../MODELS/task.model';
-import { BehaviorSubject } from 'rxjs';
+import { Task } from '../models/task.model';
+import { BehaviorSubject, map } from 'rxjs';
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Router } from '@angular/router';
+import { response } from 'express';
 
 @Injectable({
   providedIn: 'root'
 })
 export class TasksService {
+  updatedTaskList: any;
 
-  constructor() {
+  // readonly ROOT_URL = "localhost:3000" // ATM proxy handles this root
+  tasksApiUrl: string = "http://localhost:3000/tasks/"
```

Comment:
instead of defining base url here you cna defined it in a constants file where you will keep all your constants, then import the url where you need and use it how you like 😄